static/css/browser.scss

```diff
@@ -87,4 +93,9 @@ td a {
 
 .no-click circle {
   cursor: auto;
-}
\ No newline at end of file
+}
+
+.in-arc-table {
+  max-height: 450px;
```

Comment:
i'd suggest a multiple of the lineheight / table row (also specified in the same unit as lineheight - usually em)

Reply:
done